lua/lspconfig/psalm.lua

```diff
@@ -0,0 +1,28 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+local server_name = 'psalm'
+local bin_name = 'psalm-language-server'
+
+configs[server_name] = {
+  default_config = {
+    cmd = { bin_name },
+    filetypes = { 'php' },
+    root_dir = util.root_pattern('psalm.xml', 'psalm.xml.dist'),
```

Comment:
What about `composer.json`? Noticed the other pull request has it. Probably makes sense to add a fallback to `util.find_git_ancestor(fname)` as well.

Reply:
I don't think that composer or git fallback are required because the language server will not works if the psalm.xml[.dist] file doesn't exists. We will get this error  :

    $ psalm-language-server
    Could not locate a config XML file in path /home/gilles/.gbvim/. Have you run 'psalm --init' ?

If we look at the [VS Code plugin](https://github.com/psalm/psalm-vscode-plugin/blob/25d12f47f22a1a2ae18b0db17a5120c00014f0e5/src/ConfigurationService.ts#L55) and the [coc-psalm](https://github.com/yaegassy/coc-psalm), it works like this.

Reply:
Cool, nevermind then! 👍